### PR TITLE
MODSOURMAN-527 - Update dependency on processing-core library

### PR DESCRIPTION
**Pull request by RuslanLavrov**
## Purpose
it is necessary to update dependency on data-import-processing-core library to pick up fix for EDIFACT records data retrieving

## Learning
[MODSOURMAN-527](https://issues.folio.org/browse/MODSOURMAN-527)

### CODE REVIEW

**Review by Aliaksandr-Fedasiuk — APPROVED** [review]

**Comment by grabsefx** [comment]
Could we wait for release version of this dependency, not the snapshot?

**Comment by KaterynaSenchenko** [comment]
Hi @grabsefx, we need to test the changes before the release (which might happen today), but to do so on hosted ref envs - we need to update the dependency on snapshot version

**Review by KaterynaSenchenko — APPROVED** [review]

**Comment by RuslanLavrov** [comment]
Hello @grabsefx, it is necessary to verify the fix on the hosted envs and to unblock the library release. We plan to release the library in the nearest future. As soon as the library is released, I will update the dependency on the release version.

**Review by grabsefx — APPROVED** [review]

**Comment by grabsefx** [comment]
> Hi @grabsefx, we need to test the changes before the release (which might happen today), but to do so on hosted ref envs - we need to update the dependency on snapshot version

Actualy, you dont need to create PR for this purpose. You could just push your changes to the separate brunch, jenkins will build the docker container for you, and you can deploy it on any reference environment manually

**Pull request merged**
